### PR TITLE
[ELY-1691] Add test if Mechansim Database cover cipher suites from JDK

### PR DESCRIPTION
**Pull request by Skyllarr**
Jira: 
https://issues.jboss.org/browse/ELY-1691

### CODE REVIEW

File: src/test/java/org/wildfly/security/ssl/MechanismDatabaseTest.java [hunk 1]
```diff
@@ -43,4 +48,20 @@ public void testEdhDheMapping() {
         System.out.println(entry);
     }
 
+    @Test
+    public void testAllJdkCipherSuitesMapping() {
+        final MechanismDatabase mechanismDatabase = MechanismDatabase.getInstance();
+
+        SSLServerSocketFactory ssf = (SSLServerSocketFactory) SSLServerSocketFactory.getDefault();
+        String[] supportedCipherSuites = ssf.getSupportedCipherSuites();
+        List<String> TLS1_3CipherSuites = Arrays.asList("TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384");
```
Comment (fjuma) [inline src/test/java/org/wildfly/security/ssl/MechanismDatabaseTest.java]: I think it's safer to list all five of the TLS 1.3 cipher suites here (the ones that are actually available are dependent on the JDK version).

**Comment by fjuma** [comment]
@Skyllarr I think it would also be good to create an ELY issue for updating this test once support for TLS 1.3 (ELY-1706) has been added.

File: src/test/java/org/wildfly/security/ssl/MechanismDatabaseTest.java (hunk 1 above)
Comment (Skyllarr) [reply src/test/java/org/wildfly/security/ssl/MechanismDatabaseTest.java]: fixed

**Comment by fjuma** [comment]
> Done: https://issues.jboss.org/browse/ELY-1754
> (not sure whether type Bug is adequate but if not I can edit)

Thanks @Skyllarr! I think you could use "Task" for this.

**Comment by fjuma** [comment]
This is ok to test

**Pull request merged**